src/http_tracker_connection.cpp

```diff
@@ -333,7 +333,11 @@ namespace libtorrent
 
 		tracker_response resp = parse_tracker_response(data, size, ecode
 			, tracker_req().kind == tracker_request::scrape_request
-			, tracker_req().info_hash);
+			, tracker_req().info_hash
+#if TORRENT_USE_I2P
+			, tracker_req().i2pconn != NULL
```

Comment:
it would probably be better to replace the two bools (scrape_request and is_i2p) with a flags field and have two flags. That way the signature would not have to depend on TORRENT_ISE_I2P either